src/web/app/pages-session/session-result-page/session-result-page.component.html

```diff
@@ -41,7 +41,7 @@ <h1>
       <div class="card-body">
         <tm-question-text-with-info [questionNumber]="question.feedbackQuestion.questionNumber" [questionDetails]="question.feedbackQuestion.questionDetails" class="question-text"></tm-question-text-with-info>
         <tm-single-statistics [question]="question.feedbackQuestion.questionDetails"
-                              [responses]="question.responsesToSelf"
+                              [responses]="question.allResponses"
```

Comment:
At least the filtering should happen here.

Reply:
Shifted the filtering into single-statistics